app/static/styles/main.css

```diff
@@ -106,4 +106,50 @@ body {
     color: white;
     font-size: 50px;
     font-weight: 700;
+}
+
+html{
```

Comment:
I'd recommend putting all element selectors before all class selectors for consistency.